yaptide/routes/utils/response_templates.py

```diff
@@ -2,20 +2,19 @@
 from flask import make_response, Response
 
 
-def yaptide_response(message: str, code: int, content: Union[dict, str] = "") -> Response:
+def yaptide_response(message: str, code: int, content: dict = {}) -> Response:
     """Function returning Response object"""
     response_dict = {'message': message}
-    if type(content) is dict:
-        for key, value in content.items():
-            response_dict[key] = value
+    for key, value in content.items():
+        response_dict[key] = value
```

Comment:
```suggestion
    response_dict.update(content)
```

See https://docs.python.org/3.10/library/stdtypes.html#dict.update